wezterm-gui/src/glyphcache.rs

```diff
@@ -361,7 +361,7 @@ impl BlockKey {
         let c = c as u32;
         Some(match c {
             // BOX DRAWINGS LIGHT HORIZONTAL
-            0x2500 => Self::Poly(&[Poly {
+            0x2500 => Self::Poly(&[Poly { // ─
```

Comment:
I'd prefer it if the comment was on the line above the codepoint value; for the items that are labelled with the name, let's put it in with the comment; either at the LHS or the RHS of the line (RHS show in this suggestion, but I wonder if LHS might be better)

```suggestion
            // BOX DRAWINGS LIGHT HORIZONTAL  ─
            0x2500 => Self::Poly(&[Poly {
```

Reply:
Hmm was thinking about this, but it wasn't easy to automate in vim, so I made this first impl for now.
I think LHS might be better, also I'd like to surround the char with `(X)` or `[X]` to better show where the char begins/ends (useful for the partial blocks)
What do you think?

Reply:
`[X]` sounds good, and I agree: LHS feels easier to visually scan